nginx.tpl

```diff
@@ -10,6 +10,19 @@ server {
 
     include /etc/nginx/conf.d/{{ service.vhosts[0] }}/*.conf;
 
+    # From https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622
+    location ^~ /.well-known/acme-challenge/ {
+        default_type "text/plain";
+        alias /letsencrypt/well-known/;
```

Comment:
I don't think this should be hardcoded, should be passed in as an argument (sorry!)